### ISSUE TITLE
Language screen

### ISSUE THREAD

**Issue by Rajat-Sen** [StatCan/StatcanPublicationUAT#307]
In English:

1. "Language/Langue" should be read as a heading
2. English should be labelled as "Select English " button
3. French should be labelled as "Select French " button

In French: 

1. "Langue/Language" should be read as a heading 
2. It should be read as "Selectionnez Anglais" bouton
3. It should be read as "Selectionnez Francais" bouton

**Comment by ForumG-SC** [comment]
Done
